src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -804,14 +804,14 @@ def load_arguments(self, _):
     with self.argument_context('vm run-command show') as c:
         c.argument('vm_name', run_cmd_vm_name)
         c.argument('run_command_name', run_cmd_name_type)
-        c.argument('expand', help='The expand expression to apply on the operation.')
```

Comment:
In addition, please do not delete the previous parameter directly, and deprecate it to avoid breaking change to users